pkger/service.go

```diff
@@ -1032,6 +1043,18 @@ func (s *Service) Apply(ctx context.Context, orgID, userID influxdb.ID, pkg *Pkg
 		}
 	}
 
+	// this is required after first primary run and before secondary run, b/c this has dependencies
```

Comment:
Nit, but feels like this comment could be simplified / made more easy to understand.

Something like
"This has to be run after the above primary resources, because it relies on notification endpoints already being applied."

Reply:
excellent point! thank you for improving that, struggled to figure out what to write, your version is miles ahead of where mine was lol